firmware/controllers/limp_manager.cpp

```diff
@@ -140,6 +151,6 @@ LimpState LimpManager::allowIgnition() const {
 	}
 	if (!m_transientAllowIgnition) {
 		return {false, m_transientAllowIgnition.clearReason};
-	}
+	}	
```

Comment:
@opelpanfan mostly because I am a grumpy old man. Why did we need to change this line?

Please consider reviewing every line you push/PR.